aten/src/ATen/core/DeprecatedTypeProperties.h

```diff
@@ -31,7 +31,7 @@ class CAFFE2_API DeprecatedTypeProperties {
   }
 
   bool is_sparse() const {
-    return layout_from_backend(backend()) == kSparse;
```

Comment:
Renaming of `kSparse` to `kSparseCOO` is BC-breaking, and a safer way to do this change is to keep `kSparse` around as an alias to the old value (deprecated), and remove it later.

Reply:
Alright. I'll remove the commits that make this change to make the transition smoother.